spec/requests/articles_spec.rb

```diff
@@ -2,55 +2,40 @@
 require 'rails_helper'
 
 RSpec.describe ArticlesController do
-#    describe '#index' do 
-#  it 'returns a success response' do
-#  get'/articles'
- 
-#  expect(response).to have_http_status(:ok)  
-# end
-# it 'returns a proper JSON' do 
-# end
-# article= create :article
-# get '/articles'
-#testing if match with the requirements               
-# expect(json_data.length).to eq(1)
-#       expected = json_data.first
-#       #if fail then it will point out to error
-#       aggregate_failures do
-#         expect(expected[:id]).to eq(article.id.to_s)
-#         expect(expected[:type]).to eq('article')
-#         expect(expected[:attributes]).to eq(
-#           title: article.title,
-#           content: article.content,
-#           slug: article.slug
-#         )
-#       end
-#     end
-#   end
 
 it 'returns articles in the proper order' do
   older_article =
-    create(:article, created_at: 1.hour.ago)
-  recent_article = create(:article)
+    create(:article, slug: 'test1', created_at: 1.hour.ago)
```

Comment:
@Bunlong22 

must)

your model defines slug column as `uniqueness`
hence, you have to guarantee `slug` data being kept uniqueness. (duplicate data occurs test error)